app/javascript/bundles/User/components/UserShow.jsx

```diff
@@ -16,24 +16,25 @@ function UserShowView ({ user, subscription, streak }) {
   const classes = useStyles()
 
   return (
-    <Paper className={classes.root}>
-      <h3>Contact Data</h3>
-      <p>
-        <strong>Name:</strong>
-      </p>
-      <p>{user.name}</p>
-
-      <p>
-        <strong>Email:</strong>
-      </p>
-      <p>{user.email}</p>
-
-      {subscription && streak && (
+    <>
+      {subscription && subscription.active && streak && (
```

Comment:
What about using safe navigation operator here?

```ruby
subscription&.active && streak
```

Reply:
We don't have it in JavaScript (yet)

Reply:
right!, I thought this was ruby for a second 😅